### DISCUSSION TITLE
HotChocolate Mocking Server

### DISCUSSION THREAD

**Discussion by michaelstaib** [ChilliCream/hotchocolate#4844] [Feature Request]
We want to allow mocking schemas very easily. Basically you add a mocking schema by providing the sdl. 

A mocking middleware will then add some defaults. Each default can be overridden with an explicit resolver providing better defaults.

**Comment by Homberto** [comment]
I'm guessing not by the general quiet in this thread, but just for completeness I'll ask, has there been any progress on this?  I have been tinkering with the ITypeModule interface for building a schema dynamically and resolving the results from json but am having practical issues with defining the resolvers for each field.  Is a general solution for Mocking data still in the works?
